client/translations/report-client-benefits-check.en.yml

```diff
@@ -52,6 +52,13 @@ form:
             neverChecked: I've never checked this
         dateCheckedHint: Date checked
         neverCheckedHint: Please tell us why you have never checked this
+        errors:
+            noOptionSelected: Tell us if you have checked %client%'s benefits entitlement
+            missingDate: Enter the date you last checked %client%'s benefits
+            invalidDate: Enter a real date
```

Comment:
Maybe it's just me but I feel like this error message is a bit forceful? What does the date input field look like? Is it the DD/MM/YYYY format?

If so then I imagine someone entering `35/13/2015` would return the error `Enter a real date` and while the error message is correct that this is not a real date I think phrasing it like `Date invalid. Please enter a valid date.` would feel a little less forceful maybe

Reply:
Yeah that's a fair point. The content all came from Uchenna so I'd assumed it was following MoJ/GDS style approaches but will raise with Stacey to re-confirm its what we want.

Reply:
Ah fair enough yeah if it's part of the MoJ/GDS style then obviously it is what it is just felt a little forceful reading it myself 😆